core/java/iesi-core/src/main/java/io/metadew/iesi/script/action/sql/SqlExecuteStatement.java

```diff
@@ -55,8 +55,14 @@ protected boolean executeAction() throws InterruptedException {
         String sqlStatement = convertSqlStatement(getSqlStatement().getValue());
         String connectionName = convertConnectionName(getConnectionName().getValue());
         // Get Connection
-        Connection connection = ConnectionConfiguration.getInstance().get(new ConnectionKey(connectionName, getExecutionControl().getEnvName()))
-                .orElseThrow(() -> new RuntimeException("Cannot find connection " + connectionName));
+        Connection connection;
+        try {
+            connection = ConnectionConfiguration.getInstance().get(new ConnectionKey(connectionName, getExecutionControl().getEnvName()))
+                    .orElseThrow(() -> new RuntimeException("Cannot find connection " + connectionName));
+        } catch (Exception e) {
+            throw new RuntimeException("Unknown connection name");
+        }
+
```

Comment:
Why catch the generic Exception and rethrow as a Runtime Exception. A non-existing connection name would already be clear from the RuntimeExcpetion

Reply:
Corrected in next commit